### DISCUSSION TITLE
Blazor and Orchard Core with Peter Matthews - Orchard Core Pair Programming by Lombiq

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16296] [Show and tell]
Join this session of Orchard Core Pair Programming by Lombiq! In these, we do an hour of pair programming with an Orchard Core community member about a project of theirs. We learn together a lot, share best practices, and write some good code. All this is live, and you can join us with your questions! Check it out here: https://www.youtube.com/watch?v=IZioflrC1Ho See you at 5 PM UTC on the 17th!

@ApacheTech joins us this time. We'll run through using Blazor (.NET 8) as a decoupled frontend for Orchard Core. From File → New Project to production, with instructions on project setup, and a walkthrough of a fully fleshed-out site.

Your host and the "navigator" of the pair programming session will be me.

Would you like to be our guest? Just write a comment here!

**Comment by Piedone** [comment]
We're starting in about 5 minutes: https://www.youtube.com/watch?v=IZioflrC1Ho.

**Comment by Voice3** [comment]
@Piedone Watching Peter Matthews' [@ApacheTech](https://github.com/ApacheTech) session was incredibly inspiring and gave me the motivation to successfully integrate OrchardCore into my MAUI Hybrid Web Project. It was a pivotal moment that brought clarity and direction to this setup process!

If you are interested I have uploaded my repo to [https://github.com/Voice3/OrchardHybrid](https://github.com/Voice3/OrchardHybrid)

**Reply by Piedone** [reply]
I'm really glad to hear that! Thank again @ApacheTech.

**Reply by ApacheTech** [reply]
I'm happy that it was of use, and inspired people. Since then, we've gone on to use Orchard Core for tenancy onboarding, for an Azure hosted SaaS, with B2C. It's still very new, and very closed source, but it's been interesting playing around with all the ShellSettings, and SetupContexts from the point of view of Azure Groups, and getting everything to play nicely together. I'm very glad of the changes in 2.x, where we can now remove tenants programmatically.
